frontend/packages/knative-plugin/src/models.ts

```diff
@@ -32,8 +32,8 @@ export const KnativeServingModel: K8sKind = {
   apiGroup: 'operator.knative.dev',
   apiVersion: 'v1alpha1',
   kind: 'KnativeServing',
-  label: 'Knative Serving',
-  labelPlural: 'Knative Servings',
+  label: 'KnativeServing',
+  labelPlural: 'KnativeServings',
```

Comment:
No `label...Key`?

Reply:
@debsmita1 I think for translation we should follow this pattern i.e add `labelKey` and `labelPluralKey` and use the same where applicable for translations. 

https://github.com/openshift/console/blob/master/frontend/packages/pipelines-plugin/src/models/pipelines.ts#L7-L23